b64ct/src/lib.rs

```diff
@@ -123,6 +123,40 @@ pub fn decode<'a>(src: &str, dst: &'a mut [u8]) -> Result<&'a [u8], Error> {
     }
 }
 
+/// Decode B64-encoded string in-place.
+pub fn decode_inplace(buf: &mut [u8]) -> Result<&[u8], InvalidEncodingError> {
```

Comment:
For consistency (with e.g. [AeadInPlace](https://docs.rs/aead/0.3.2/aead/trait.AeadInPlace.html)) I'd suggest calling this `decode_in_place`